### DISCUSSION TITLE
Auto-Type database selection

### DISCUSSION THREAD

**Discussion by bergsj** [keepassxreboot/keepassxc#6871] [Ideas]
Hi,

Loving the work on KeePassXC first of all!

What I would like to suggest is a feature within Auto-Type that when KP runs minimized, but has your databases locked (after locking the Windows session or when minimized), it will ask you which database to unlock. 
Currently when you have multiple databases and KP is locked, Auto-Lock will ask you to unlock the latest database you've used, but when you want to Auto-Type a password from another database, you first need to open KP, unlock that database then go back to the application and press the Auto-Lock shortcut again. A separate popup to select which database to unlock would then be helpful and make things a little bit more efficient.

What do you guys think?

Cheers
Sjoerd

**Comment by phoerious** [comment]
This is planned and I believe there is an issue somewhere, but I can't find it right now.

**Comment by droidmonkey** [comment]
#5427

**Comment by bergsj** [comment]
Thanks for the feedback!
